gh2/csv.py

```diff
@@ -56,8 +56,13 @@ def make_parser():
         help='Only include issues with this label in the output. Multiple '
              'invocations of this flag will require the issue to have all '
              ' labels.',
-        action='append', dest='filter_labels',
-        default=[],
+        action='append', dest='filter_labels', default=[],
+    )
+    args.add_argument(
```

Comment:
How common is filtering by milestone? Does this need to be optional?

Without knowing how this tool is currently being used, I cannot say for sure, but it strikes me that this piece of data would be wanted in the output more often than it wasn't. If it's preferable to keep it optional, maybe it should be included by default.